docs/source-pytorch/fabric/guide/checkpoint.rst

```diff
@@ -0,0 +1,82 @@
+:orphan:
+
+##############################
+Saving and Loading Checkpoints
+##############################
+
+Fabric makes it easy and efficient to save the state of your training loop into a checkpoint file, no matter how large your model is.
+
+----
+
+
+********************************
+Define the state of your program
+********************************
+
+In order save and resume your training, you need to define which variables in your program you want to have saved.
+Put everything into a dictionary, including models and optimizers and whatever metadata you have:
+
+.. code-block:: python
+
+    # Define the state of your program/loop
+    state = {
+        "model1": model1,
+        "model2": model2,
+        "optimizer": optimizer,
+        "iteration": iteration,
+        "hparams": ...
+    }
+
+
+----
+
+
+*****************
+Save a checkpoint
+*****************
+
+To save the state to the filesystem, pass it to the :meth:`~lightning_fabric.fabric.Fabric.save` method:
+
+.. code-block:: python
+
+    fabric.save("path/to/checkpoint.ckpt", state)
+
+This will unwrap your model and optimizer and convert their `state_dict` automaticall for you.
+Fabric and the underlying strategy will decide in which format your checkpoint gets saved.
+For example, ``strategy="ddp"`` saves a single file on rank 0, while ``strategy="fsdp"`` saves multiple files from all ranks.
+
+
+----
+
+
+*************************
+Restore from a checkpoint
+*************************
```

Comment:
These are very basic docs.

Once we have the deepspeed and fsdp implementations, I will create a document linked here that explains best practices and efficiency around saving and loading large sharded models.